src/en/news/blog/2022/dashboard-rbd-scaling/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Dashboard scability of RBD images"
+date: "2022-09-01"
+author: "Pere Diaz Bou"
+image: "images/image.jpg"
+tags:
+  - "mgr"
+  - "performance"
+  - "python"
+  - "rbd"
+---
+
+## Overview
+
+Management of RBD images in the Dashboard module has been challenging due to its low performant
+behaviour with a high enough number of images. With enough images tables would see long loading
+times which might seem infinite rendering them unusable.
+
+After introducing server side pagination with RBD images we've been able to see a clear and expected
+**reduction of around 99%** in latency.
+
+## Tradeoffs
+
+Introducing pagination didn't come without tradeoffs. After running some experiments we noticed that
+retrieving all data of an image could take a considerable amount of time ranging from 10ms-30ms,
+meaning that gathering all images to later paginate is impossible. Luckily, we can gather all of the
+references to images in a short amount of time, for example, 2048 references were retrived in 10ms.
+
+**Sorting and searching images is now limited to image names, pools and namespaces** because of the
+limitations of these references, but the management of RBD images is now available to use at greater
+scale.
```

Comment:
I'd talk about future plans to paginate other components.

Reply:
Ok